### PR TITLE
Adding fan Animation himanshu-sheetlani

### PR DESCRIPTION
**Pull request by himanshu-sheetlani**
Fan Animation By `https://github.com/himanshu-sheetlani`

### CODE REVIEW

File: Art/himanshu-sheetlani/Fan Animation.gif
Comment (LaurelineP) [inline Art/himanshu-sheetlani/Fan Animation.gif:1]: ## Image / pictural representation 
The gif is correct  but the file name contains a space - a special character which is not valid as it is then transformed as %20.
Please rename your file name so it contains valid characters

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha @himanshu-sheetlani  🙌
Thanks for your contribution and time !
Before we could merge your contribution, the following feedbacks need to be addressed first.
Please address the given feedbacks

> [!NOTE]
Checked values - are the one that are correctly done
> - Please address all the unchecked elements
> - If you need guidance , please refer to the

> [!TIP]
> If the name of your folder needs to be changed ensure to adjust the `include.js`

## Feedbacks 
- Javascript is not allowed to animate 
- do not forget to retake an record of your animation once js is removed
- do not forget to provide a valid filename to your gif
- do not forget to review your imageUrl in include.js
when you would rename the gif file
## 🔹 Reference Acceptance criteria for a contribution
Correctly:
  - [x] Folder name: named after the github user name in `./Art`  
     - [x] contains a `HTML` file: the contribution must have an HTML file
         - [x] has a <link> tag to the CSS file of the project
         - [x] contains the `HTML` content for the animation
         - [x] does not contain a SVG replacing the whole HTML CSS animation requirements
     - [x] contains a `CSS` file: the contribution must have a CSS file 
     - [ ] does **not** contain a `JS` file or JS logic : No JS logic should be part of your submission
     - [x] the contribution includes the animated file (gif) or picture ( jpeg, jpg, png )
     - [x] the pictural representation must not include the browser frame, just the animation
     - [x] the contribution includes the correct contribution's details in `include.js`
        - [x]  the relative path should be correct : referring to the correct named folder
        - [x] `author`: the value should be the github name or github
        - [x] `artName`: the value should be the animation title
        - [x] `pageLink`: the value should  be the relative path to your project's `HTML` file
        - [x] `imageLink`: should refer to the project relative path's image 
        - [x] `githubLink`: the value should be your github profile link
        - [x] does not have chevrons
        - [x] does not have comments for each fields

File: Art/himanshu-sheetlani/Fan Animation.gif
Comment (himanshu-sheetlani) [reply Art/himanshu-sheetlani/Fan Animation.gif:1]: Hello, @LaurelineP. I saw all your reviews and I have made all the necessary changes.
changes made:
1. Removed JavaScript from animation and made it non-interactive.
2. Added few more elements to make it look better.
3. Removed spaces in the name from the gif file.
Thanks for reaching out to me about changes.

File: include.js
Comment (LaurelineP) [inline include.js:1]: ## Incorrect `imageLink`
Your current `imageLink` represent the previous link ( also it had a space )
it should be your new gif instead ( which is different ) 
Please adjust the imagelink :)

**Review by LaurelineP — CHANGES_REQUESTED** [review]

File: include.js
Comment (himanshu-sheetlani) [reply include.js:1]: @LaurelineP Done

**Pull request merged**